src/core/test/system/lender/redeem.t.sol

```diff
@@ -20,26 +20,69 @@ import {SwitchableDistributor} from "../../../lender/distributor/switchable.sol"
 
 contract RedeemTest is SystemTest {
```

Comment:
maybe create a switchable_distributor folder in lende rand put all the related tests there